index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+    <div class="center">
+        <h2>freeCodeCamp Survey Form</h2>
+        <p>Thank you for taking the time to help us improve the platform</p>
+    </div>
+    <div class="form-body">
+        <form class="input" action="submit">
```

Comment:
Please add the `method` attribute if this form is designed to post and get data.

Reply:
- [ ] Nice job asking for changes in a friendly and polite way and using backticks to highlight the attribute name but you forgot to mention the reason why he had to do this change and you can provide the reviewee with a link to make him understand better why he consider adding the method attribute like this [article](https://html.com/attributes/form-method/#:~:text=%3Cform%20method%3D%E2%80%9DPOST%E2%80%9D%3E%20The%20method%20attribute%20of%20the%20form,data%20to%20the%20web%20server%20to%20be%20processed).

Reply:
> * [ ]  Nice job asking for changes in a friendly and polite way and using backticks to highlight the attribute name but you forgot to mention the reason why he had to do this change and you can provide the reviewee with a link to make him understand better why he consider adding the method attribute like this [article](https://html.com/attributes/form-method/#:~:text=%3Cform%20method%3D%E2%80%9DPOST%E2%80%9D%3E%20The%20method%20attribute%20of%20the%20form,data%20to%20the%20web%20server%20to%20be%20processed).

Please add the `method` attribute if this form is designed to post or get data. The value of this attribute specifies how the browser will send the form data to the webserver. Kindly read this [article](uhttps://html.com/attributes/form-method/#:~:text=%3Cform%20method%3D%E2%80%9DPOST%E2%80%9D%3E%20The%20method%20attribute%20of%20the%20form,data%20to%20the%20web%20server%20to%20be%20processedrl) for more information.